examples/user_checker.rb

```diff
@@ -40,7 +40,7 @@ class UserChecker
 
   options sidekiq: { queue: :fast }
 
-  notify "tech@drivy.com"
+  notify :action_mailer, target: "tech@drivy.com"
```

Comment:
Do you think a `to:` options would make sense? Could accept other keywords such as `from:` to override some of the existing configuration too?